Gemfile

```diff
@@ -30,4 +37,10 @@ group :development do
   gem 'web-console', '>= 3.3.0'
 end
 
+group :test do
+  gem 'database_cleaner', '~> 1.5'
+  gem 'faker', '~> 1.6.1'
+  gem 'shoulda-matchers', '~> 3.0', require: false
+end
```

Comment:
again the game with the gems... In one PR someone is removing them, in other someone is putting them back... 
Why? https://github.com/rokor/Ruby_Combats/pull/24/files